ui/src/Pages/SettingsPage/UserPreferences.tsx

```diff
@@ -68,7 +68,7 @@ const languagePreferences = [
   {
     key: 15,
     label: 'Українська',
-    flag: 'ua',
+    flag: 'uk',
```

Comment:
This `uk` is conflicted with the first entity `English`/`uk`.

I don't know which codes we are using here. In ISO 639-1, it should be `en` in `English` and `uk` in `Ukrainian`. In ISO 3166-1 alpha-2, it should be `GB` and `UK`.

Since it's language, I guess we should update `English` as `en` too.
```suggestion
    flag: **'uk',**
```
```suggestion
    flag: **'uk',**
```